### DISCUSSION TITLE
手表协议下，不能获取到群里发的视频嘛

### DISCUSSION THREAD

**Discussion by jiangdengke** [simple-robot/Discussions#152] [Q&A] [answered]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

想把群里的视频存起来，发现手表协议下好像，视频属于文本消息，日志是这样的：
[文本消息: 你的QQ暂不支持查看视频短片，请期待后续版本。 ]

**Comment by ForteScarlet** [comment, marked as answer]
手表应该是看不了视频的，自然也就接收不了

**Reply by jiangdengke** [reply]
好的
